core/server/models/base/token.js

```diff
@@ -36,7 +36,7 @@ Basetoken = ghostBookshelf.Model.extend({
             .query('where', 'expires', '<', Date.now())
             .fetch(options)
             .then(function then(collection) {
-                collection.invokeThen('destroy', options);
+                return collection.invokeThen('destroy', options);
```

Comment:
i didn't concentrated on verifying that, just stumbled over the non existing return. 
i can extend my big test case to ensure expired tokens get deleted?maybe a good idea